pages/me.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { useRouter } from 'next/router';
+import {LINKEDIN_OAUTH} from '../pages/api/linkedin'
+
+const Me = () => {
+  const router = useRouter(); 
+  if (router.query.code) {  
+  LINKEDIN_OAUTH(router.query.code)
```

Comment:
this is nitpick: suggestion to align it properly - add another space

Reply:
i fixed the spacing in the next commit :)